tests/src/test/scala/neotypes/AsyncSuite.scala

```diff
@@ -1,12 +1,10 @@
 package neotypes
 
-import org.scalatest.Suites
-
 import scala.concurrent.{ExecutionContext, Future}
 import scala.reflect.ClassTag
 
 /** Testkit used to write specs abstracted from any concrete Async type. */
-abstract class AsyncTestkit[F[_]](implicit ct: ClassTag[F[_]]) {
+abstract class AsyncTestkit[F[_]](implicit ct: ClassTag[F[AnyRef]]) {
```

Comment:
To be honest, I'm not sure this is 100% safe.

Reply:
scattering type params means this approach. https://github.com/neotypes/neotypes/pull/667#discussion_r1307607994